docs/modules/ROOT/pages/scalalib/native-examples.adoc

```diff
@@ -0,0 +1,30 @@
+= Scala Native Examples
+:page-aliases: Scala_Native_Examples.adoc
+
+++++
+<script>
+gtag('config', 'AW-16649289906');
+</script>
+++++
+
+
+This page contains examples of using Mill as a build tool for scala-native applications.
+It covers setting up a basic basic scala-native application that calls C function within it,
+as well as example of two modules with a scala-naive application.
```

Comment:
s/naive/native/g

Reply:
Please what you meant by this **s/native/native/g**?

Reply:
Oh... Maybe you meant the double in the text **... basic basic scala-native application ...**

Reply:
I meant the typo, `naive` should be `native`. But yeah you need to check your english prose before passing to me to review

Reply:
Just spot this other typo